<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ZSCHN01LP0340" failures="1" tests="2" name="Test.EmployeeDownloadTest$TestUserCreation" time="8.277" errors="0" timestamp="2022-12-26T21:11:52 IST" skipped="0">
  <testcase classname="Test.EmployeeDownloadTest$TestUserCreation" name="userCreation" time="6.231"/>
  <system-out/>
  <testcase classname="Test.EmployeeDownloadTest$TestUserCreation" name="logoutFromApplication" time="2.046">
    <failure message="Unable to logout from the application" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Unable to logout from the application
at org.testng.Assert.fail(Assert.java:99)
at Pages.webAction.logoutFromApplication(webAction.java:157)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)
at org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)
at org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)
at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:217)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:130)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at Test.TestNGXML.xml(TestNGXML.java:116)
at Test.TestNGXML.main(TestNGXML.java:121)
]]>
    </failure>
  </testcase> <!-- logoutFromApplication -->
  <system-out/>
</testsuite> <!-- Test.EmployeeDownloadTest$TestUserCreation -->
